set.mm

```diff
@@ -759848,15 +759848,15 @@ this concept is not fully developed yet (it is within a mathbox), and it is
 
 $)
 
-  $c NaryF $.  $( Constant symbol for the n-ary functions. $)
+  $c -aryF $.  $( Constant symbol for the n-ary functions. $)
 
   $( Extend the definition of a class to include the n-ary functions. $)
-  cnaryf $a class NaryF $.
+  cnaryf $a class -aryF $.
 
   ${
     $d n x $.
     $( Define the n-ary (endo)functions.  (Contributed by AV, 11-May-2024.) $)
-    df-naryf $a |- NaryF = ( x e. _V , n e. NN0
```

Comment:
@icecream17 if you like, you can add a "Revised by" tag here with your and/or @tirix's name ,